x-pack/plugins/ml/public/embeddables/single_metric_viewer/single_metric_viewer_embeddable_factory.tsx

```diff
@@ -147,31 +123,15 @@ export const getSingleMetricViewerEmbeddableFactory = (
         services[1].data.query.timefilter.timefilter
       );
 
+      const SingleMetricViewerComponent = getSingleMetricViewerComponent(
+        services[0],
+        services[1],
+        services[2]
+      );
```

Comment:
```suggestion
      const SingleMetricViewerComponent = getSingleMetricViewerComponent(...services);
```

Reply:
Updated in 6c61014b8a7ef4a3167a642e5f72f1938ea25c50